cql3/expr/expression.cc

```diff
@@ -185,6 +193,18 @@ bool equal(const bytes_opt& rhs, const column_value& lhs, const column_value_eva
     return get_value_comparator(lhs)->equal(*value, *rhs);
 }
 
+/// True iff lhs's value equals rhs.
+bool equal(const managed_bytes_opt& rhs, const column_value& lhs, const column_value_eval_bag& bag) {
```

Comment:
I love expression.cc's convention of putting rhs on the left, and lhs on the right.